irc/handlers.go

```diff
@@ -3097,6 +3097,203 @@ func markReadHandler(server *Server, client *Client, msg ircmsg.Message, rb *Res
 	return
 }
 
+// METADATA <target> <subcommand> [<and so on>...]
+func metadataHandler(server *Server, client *Client, msg ircmsg.Message, rb *ResponseBuffer) (exiting bool) {
+	if !server.Config().Metadata.Enabled {
+		return unknownCommandHandler(server, client, msg, rb)
```

Comment:
`FAIL METADATA FORBIDDEN #baz :Metadata is disabled on this server` is probably less confusing, honestly --- with the current behavior, I anticipate a bunch of bug reports like "I'm running 2.17 but METADATA returns 'unknown command'" :-)